pipelines.adoc

```diff
@@ -4,20 +4,20 @@ permalink: /guides/pipelines-getting-started/
 :page-layout: guide
 :page-duration: 30 minutes
 :page-releasedate: 2019-10-17
-:page-description: Explore how to use Kabanero Pipelines to build and manage Kabanero collections.
+:page-description: Explore how to use Pipelines with Collections 
 :page-tags: ['Pipelines, Collection']
 :page-guide-category: pipelines
 = Build and deploy applications with Kabanero Pipelines
 
-Kabanero uses link:https://github.com/tektoncd/pipeline/tree/master/docs#usage[Tekton Pipelines] to illustrate a continuous input and continuous delivery (CI/CD) workflow. Each of the featured link:https://github.com/kabanero-io/collections[Kabanero collections] contains a default pipeline that builds the collection, publishes the image to a Docker repo, and then deploys the application to the Kubernetes cluster. You can also create your own tasks and pipelines and add them to the collections. All tasks and pipelines are activated by the link:https://github.com/kabanero-io/kabanero-operator[Kabanero operator].
+Kabanero uses link:https://github.com/tektoncd/pipeline/tree/master/docs#usage[Tekton Pipelines] to illustrate a continuous input and continuous delivery (CI/CD) workflow. Each of the featured link:https://github.com/kabanero-io/collections[ Collections] contains a default pipeline that builds the collection, publishes the image to a container registry, and then deploys the application to the Kubernetes cluster. You can also create your own tasks and pipelines and customize the pre-built pipelines and tasks.   All tasks and pipelines are activated by  link:https://github.com/kabanero-io/kabanero-operator[Kabanero's standard Kubernetes Operator].
```

Comment:
remove extra space between "tasks." and "All tasks and pipelines...."

`the pre-built pipelines and tasks.   All tasks and pipelines are activated by`